staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -951,9 +951,7 @@ func (qs *queueSet) boundNextDispatchLocked(queue *queue) {
 	}
 	var virtualStartBound = oldestReqFromMinQueue.arrivalR
 	if queue.nextDispatchR < virtualStartBound {
-		if klogV := klog.V(4); klogV.Enabled() {
```

Comment:
Working on benchmarking this function. Will update once i have the details here.

Reply:
I had no success in getting a clean way to write a benchmark test for this function. Any suggestion on this @pohly ?

Reply:
If the SIG doesn't already have benchmarks, then writing one is hard because it depends on domain knowledge.

When in doubt, leave the if check in place.

Reply:
Ack